bugbug/models/invalid_compatibility_report.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import logging
+
+import xgboost
+from sklearn.compose import ColumnTransformer
+from sklearn.pipeline import Pipeline
+
+from bugbug import feature_cleanup, issue_features, utils
+from bugbug.model import IssueModel
+
+logger = logging.getLogger(__name__)
+
+
+class InvalidCompatibilityReportModel(IssueModel):
+    def __init__(self, lemmatization=False):
+        super().__init__(
+            owner="webcompat", repo="web-bugs", lemmatization=lemmatization
+        )
+
+        self.calculate_importance = False
+
+        feature_extractors = []
+
+        cleanup_functions = []
+
+        self.extraction_pipeline = Pipeline(
+            [
+                (
+                    "report_extractor",
+                    issue_features.IssueExtractor(
+                        feature_extractors, cleanup_functions, rollback=False
+                    ),
+                ),
+                (
+                    "union",
+                    ColumnTransformer(
+                        [
+                            (
+                                "first_comment",
+                                self.text_vectorizer(min_df=0.0001),
+                                "first_comment",
+                            ),
+                        ]
+                    ),
+                ),
+            ]
+        )
+
+        self.clf = xgboost.XGBClassifier(n_jobs=utils.get_physical_cpu_count())
+
+    def items_gen(self, classes):
+        # Do cleanup separately from extraction pipeline to
+        # make sure it's not applied during classification due to differences
+        # in text structure between GitHub issues and reports
+        cleanup_function = feature_cleanup.CleanCompatibilityReportDescription()
+
+        for issue, classes in super().items_gen(classes):
+            issue = {
+                **issue,
+                "body": cleanup_function(issue["body"]),
+            }
+            yield issue, classes
```

Comment:
It is the class for a single issue. The variable `classes` is used for the classes of all issues.
```suggestion
        for issue, class in super().items_gen(classes):
            issue = {
                **issue,
                "body": cleanup_function(issue["body"]),
            }
            yield issue, class
```

Reply:
My IDE showed an error since class is a reserved name, so I renamed it to classes without realizing. So maybe I can change it to `cls

Reply:
Ooh yes! `cls` or `label` are suitable options.

Reply:
ok, thanks, I've changed it to `label`